src/main/webapp/app/shared/user-settings/ssh-settings/ssh-user-settings.component.html

```diff
@@ -0,0 +1,81 @@
+<h1>
+    <!--Account Information-->
+    {{ 'artemisApp.userSettings.sshSettings' | artemisTranslate }}
+</h1>
+@if (currentUser) {
+    <div class="list-group d-block">
+        <div class="list-group-item">
+            <dt>
+                {{ 'artemisApp.userSettings.sshSettingsPage.sshSettingsInfoText' | artemisTranslate }}
+                <a href="https://docs.artemis.cit.tum.de/user/exercises/programming.html#repository-access">
```

Comment:
Could you use the existing documentation button component for this?

Reply:
Ah that's a good tip, didn't know it existed :+1: I'll add it